resources/testdata/glyphs3/MetaTable.glyphs

```diff
@@ -0,0 +1,46 @@
+{
+.appVersion = "3260";
+.formatVersion = 3;
+DisplayStrings = (
+" "
+);
+customParameters = (
+{
+name = "meta Table";
+value = (
+{
+data = "latn-en";
+tag = dlng;
+},
+{
+data = "latn,derp";
+tag = slng;
+}
+);
+}
+);
+date = "2024-12-03 00:05:48 +0000";
+familyName = UnusualParams;
+fontMaster = (
+{
+id = master01;
+name = Regular;
+}
+);
+glyphs = (
+{
+glyphname = space;
+lastChange = "2024-12-03 00:05:51 +0000";
+layers = (
+{
+layerId = master01;
+width = 600;
+}
+);
+unicode = 32;
+}
+);
+unitsPerEm = 1000;
+versionMajor = 1;
+versionMinor = 1;
+}
```

Comment:
this can be trimmed further, for example I don't believe you need appVersion, DisplayStrings, date, lastChange of glyph, possibly width of glyph, and version major/minor